.circleci/config.yml

```diff
@@ -91,7 +91,7 @@ jobs:
 
   build:
     <<: *DOCKER_NODE
-    resource_class: small
+    resource_class: medium
     steps:
```

Comment:
Had to add more resource because of memory error:

<img width="1373" alt="Screenshot 2022-11-07 at 14 30 23" src="https://user-images.githubusercontent.com/2486360/200340535-725beaeb-a261-48be-afd4-d8178b97c8e6.png">